### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by ChrSacher**
Bugfix to:
Shotcount event handler breaking
Capturezone params
Hostage ACE call, dead hostage bug
Added hostage to config
Acre depending on fw settings

### CODE REVIEW

File: uo_sys_framework/ShotCountModule/cfgXEH/init/ServerPreInit.sqf [hunk 1]
```diff
@@ -10,6 +10,22 @@ UO_FW_EXEC_CHECK(SERVER);
...
+    _unit = _this;
+    _unit call FUNC(aCount_addEH);
+}] call CBA_fnc_addEventHandler;
+
+["UO_FW_Core_VehicleSpawnedEvent", 
+{
+    if !(UO_FW_Server_ShotCountModule_Allowed) exitwith {};
+    if !(GETMVAR(ShotCount_Enabled,false)) exitwith {};
+   _vehicle = _this;
+   _vehicle call FUNC(aCount_addEH);
```
Comment (PiZZAD0X) [inline uo_sys_framework/ShotCountModule/cfgXEH/init/ServerPreInit.sqf]: ShotCount is added through `Extended_InitPost_Eventhandlers` with ```class UO_FW_ShotCount_InitPost {
    init = "_this call UO_FW_fnc_aCount_addEH";
};```

File: uo_sys_framework/config.cpp [hunk 2]
```diff
@@ -307,6 +307,7 @@ class Extended_PreInit_EventHandlers {
 	#include "BriefingModule\cfgXEH\PreInit.hpp"
 	#include "AntiNDModule\cfgXEH\PreInit.hpp"
 	#include "TeamColourModule\cfgXEH\PreInit.hpp"
+    #include "HostageModule\cfgXEH\PreInit.hpp"
```
Comment (PiZZAD0X) [inline uo_sys_framework/config.cpp]: seriously... lol

File: uo_sys_framework/HostageModule/cfgXEH/init/ClientPreInit.sqf [hunk 3]
```diff
@@ -4,6 +4,6 @@ UO_FW_EXEC_CHECK(CLIENT);
 
 ["UO_FW_Hostage_ACEActions_GlobalEvent", {
     params ["_unit"];
-    private _rescueaction = ["Rescue Hostage", "Rescue Hostage","",{_this call UO_FW_fnc_HostageRescue},{!(GETVAR((_this select 0),Rescued,false)) && {(alive (_this select 0))}}, "", [_unit]] call ace_interact_menu_fnc_createAction;
+    private _rescueaction = ["RescueHostage", "Rescue Hostage","",{_this call UO_FW_fnc_HostageRescue},{!(GETVAR((_this select 0),Rescued,false)) && {(alive (_this select 0))}},{}, "", [_unit]] call ace_interact_menu_fnc_createAction;
```
Comment (PiZZAD0X) [inline uo_sys_framework/HostageModule/cfgXEH/init/ClientPreInit.sqf]:
```suggestion
    private _rescueAction = ["RescueHostage", "Rescue Hostage","",{[_this] call UO_FW_fnc_HostageRescue},{!((_this select 0) getvariable ["UO_FW_Hostage_IsRescued",false]) && {(alive (_this select 0))}},{}, "", [_unit]] call ace_interact_menu_fnc_createAction;
```

Comment (PiZZAD0X) [reply uo_sys_framework/HostageModule/cfgXEH/init/ClientPreInit.sqf]: Doesn't like the macro in the ace action condition

**Comment by thecogdoctor** [comment]
When going to select unit attributes to try and assign ACRE attributes the ACRE attribute once, but every subsequent attempt after that no units displayed the ACRE Options tab.

**Comment by PiZZAD0X** [comment]
ACRE attributes only available on playable units

**Pull request merged**